lib/govuk_test.rb

```diff
@@ -11,10 +11,16 @@ def self.configure(options = {})
     chrome_options << "--window-size=#{options[:window_size]}" if options[:window_size]
 
     if ENV['GOVUK_TEST_USE_SYSTEM_CHROMEDRIVER']
-      # Use the installed chromedriver, rather than chromedriver-helper
+      # Use the installed chromedriver, rather than webdrivers
       Selenium::WebDriver::Chrome.driver_path = File.which("chromedriver")
     else
-      require 'chromedriver-helper'
+      require 'webdrivers'
```

Comment:
The need for this seems to be related to the delayed invocation of the line that does the find/update for chromedriver. If we run this line manually, could we avoid the breaking change?

Reply:
Not sure I understand Ben - would you mind explaining a little more please?

Reply:
I think in the end we decided it would be better for `govuk_test` to configure WebMock anyway since this Gem is responsible for setting up the testing environment our apps use.

The only thing is that I think we should move the call to `disable_net_connect!` outside the `if` statement, so it's always called no matter whether we're using system or `webdrivers` chromehelper.

Reply:
@benthorner do you mean we can do `require 'webdrivers' unless ENV['GOVUK_TEST_USE_SYSTEM_CHROMEDRIVER']`, and we do not need to stipulate the driver_path, as Selenium will be able to locate this in anycase?

Reply:
@bevanloon so I think the problem with WebMock is coming from the webdrivers trying to lazy-check/download a new version of chromedriver after WebMock has been setup. Once webdrivers has done this once, it doesn't do it again. So if we force webdrivers to do the check/download before we setup WebMock, then there's no need to change the settings.

Reply:
I've now had some time to play myself, and can confirm the following works. We do need to make sure it comes before any call to `disable_net_connect`, and because these lines don't actually configure anything, it seems reasonable to achieve this by doing them outside of `.configure`.

```
require "selenium-webdriver"

unless ENV['GOVUK_TEST_USE_SYSTEM_CHROMEDRIVER']
  require 'webdrivers'

  # This stop webdrivers doing a lazy check for new versions
  # of chromedriver, which interferes with WebMock.disable_net_connect
  Selenium::WebDriver::Chrome.driver_path
end

module GovukTest
  def self.configure(options = {})
    chrome_options = %w(headless disable-gpu)
    chrome_options << "--window-size=#{options[:window_size]}" if options[:window_size]

    #if defined?(WebMock)
      #WebMock.disable_net_connect!(
        #allow_localhost: true,
        #allow: 'chromedriver.storage.googleapis.com'
      #)
    #end
```

This should mean we can avoid a breaking change, although it still sounds like a good idea to abstract the setup of WebMock into this gem.

Reply:
That looks like a good idea for this PR and we can look at configuring WebMock in a different one.